src/lib/helpers/load.ts

```diff
@@ -21,7 +21,7 @@ export enum View {
 export function getView(url: URL, route: Page['route'], fallback: View): View {
     return (url.searchParams.get('view') ?? preferences.get(route).view) === View.Grid
         ? View.Grid
-        : View.Table ?? fallback;
+        : (View.Table ?? fallback);
```

Comment:
I think this changes logic. Fallback will never be reached.

Reply:
Leftover from `pnpm format`. Merging main should be fine.